draft/oss-support/post.md

```diff
@@ -0,0 +1,70 @@
+# OSS Support
+
+Free software. It's not _really_ free. Time, effort, innovation and labor go
+into developing the open source projects that we use and sometimes can't live
+without. A lot of these projects live in the background, known only to those
+that dig into the low level building blocks of the net or in the lower depths
+of the application layer of open and closed operating systems (see the EULA of
+OSX and you'll see open source projects listed). Many open source projects find
+funding through sales of support, licenses, and by hosting. But quite a few are
+just put out there and sometimes they find there way into the tool boxes of
+devs and designers, to the point where they almost become ubiquitous.
+
+When you think about it, how many of the open source projects that you use all
+the time are not heavily funded operations? VI, Emacs, heck even small tools
+like sed to niche languages like tcl or lua. These are all entrenched and still
```

Comment:
Please add links to tcl and lua. Also, fin their proper names (capitalization-style) and use those.